runtime/lua/vim/lsp.lua

```diff
@@ -143,6 +143,8 @@ local function for_each_buffer_client(bufnr, fn, restrict_client_ids)
   end
 end
 
+local client_errors_offset = table.maxn(lsp.rpc.client_errors)
```

Comment:
An even nicer refactor would be making this a function and the offset an outer variable that's incremented by the utility function ;)